File: app/src/main/java/com/song/deviceinfo/ui/wifi/WifiViewModel.java
package com.song.deviceinfo.ui.wifi;

import android.content.Context;

import com.song.deviceinfo.info.WifiInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/9/22.
 */
public class WifiViewModel extends NormalViewModel {

    public List<Pair<String, String>> getWifiInfo(Context context) {
        return WifiInfo.getWifiInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/wifi/WifiViewModel.kt
package com.song.deviceinfo.ui.wifi

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.WifiInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/9/22.
 */
class WifiViewModel : NormalViewModel() {
    fun getWifiInfo(context: Context): List<Pair<String, String>> {
        return WifiInfo.getWifiInfo(context)
    }
}
